tools/getkeys/.eslintrc.cjs

```diff
@@ -9,7 +9,7 @@ module.exports = {
 		"prettier",
 	],
 	parserOptions: {
-		project: ["./tsconfig.json"],
+		project: ["./tsconfig.lint.json"],
```

Comment:
Why?

Reply:
Because it is only used for linting - there is no ts in here and lint config makes packages have a tsconfig, Adding .lint to tsconfig is recognized by policy checks as not a real tsconfig to be used.